.github/workflows/freshdesk-connector-ci.yaml

```diff
@@ -0,0 +1,58 @@
+name: freshdesk-connector CI
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+env:
+  REGISTRY: docker.io
+  IMAGE_NAME: ${{ github.actor }}/${{ github.event.repository.name }}
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/main' || startsWith(github.ref, 'refs/tags')
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run Unit Tests
+        run: go test
```

Comment:
I assume this will fail because of no go tests at the moment. Correct?

Maybe we can add a single test for now, and build from there.

Reply:
Sure, let me update with some sample tests.

Reply:
Integrated with sample unit tests.